rules/falco_rules.yaml

```diff
@@ -110,7 +110,7 @@
 
 # This detects writes immediately below / or any write anywhere below /root
 - macro: root_dir
-  condition: ((fd.directory=/ or fd.name startswith /root) and fd.name contains "/")
+  condition: ((fd.directory=/ or fd.name startswith /root/) and fd.name contains "/")
```

Comment:
can you remove `and fd.name contains "/"` -- this part is redundant to me.

Reply:
I did not notice that! Thank you!

Fixed by https://github.com/falcosecurity/falco/pull/1279/commits/5f0d1b832a0a6b689050a96938b3b9ef8b24cadb